androidHyperskillApp/src/main/java/org/hyperskill/app/android/home/view/ui/fragment/HomeFragment.kt

```diff
@@ -129,17 +139,21 @@ class HomeFragment :
     override fun onDestroyView() {
         super.onDestroyView()
         gamificationToolbarDelegate = null
+        problemsLimitDelegate?.cleanup()
```

Comment:
ProblemsLimitDelegate holds a reference to the viewBinding, that's why it should be destroyed here, so that view reference will not be held after onDestroyView hook.